src/orb.yml.hbs

```diff
@@ -132,6 +132,31 @@ jobs:
           "The name of the service to update. If undefined, we assume `family` is the name of both the service and task definition."
         type: string
         default: ""
+      deployment-controller:
+        description:
+          "The deployment controller to use for the service. Defaulted to ECS"
+        type: string
```

Comment:
I think it would be great if the type could be an enum: https://circleci.com/docs/2.0/reusing-config/#enum

Reply:
Fixed by 4da210a .